go-zen-chu/nuc-playbook
# seperated k8s manifests & update blackbox-exporter config

## Why

- blackbox exporter was not working
- every time you apply k8s manifest, you needed to run ansible-playbook

## What

- separated k8s manifest
- configured blackbox exporter with ansible variable (to configure nfs, ssh port……)

## QA

```
$ ansible-playbook -i hosts.yml site.yml --tags k8s                                                                                                                            (git)-[seperate-k8s-manifest] 
SSH password: 

PLAY [all] *******************************************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] *******************************************************************************************************************************************************************************************************************************************
[WARNING]: Platform linux on host nuc is using the discovered Python interpreter at /usr/bin/python, but future installation of another Python interpreter could change the meaning of that path. See
https://docs.ansible.com/ansible/2.10/reference_appendices/interpreter_discovery.html for more information.
ok: [nuc]

TASK [k8s : copy k8s manifests] **********************************************************************************************************************************************************************************************************************************
ok: [nuc]

TASK [k8s : generate prometheus configmap] ***********************************************************************************************************************************************************************************************************************
ok: [nuc]

TASK [k8s : apply k8s setup manifest] ****************************************************************************************************************************************************************************************************************************
changed: [nuc]

TASK [k8s : apply k8s ingress manifest] **************************************************************************************************************************************************************************************************************************
changed: [nuc]

TASK [k8s : apply k8s monitoring manifest] ***********************************************************************************************************************************************************************************************************************
changed: [nuc]

PLAY RECAP *******************************************************************************************************************************************************************************************************************************************************
nuc                        : ok=6    changed=3    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

```
